jwst/resample/resample.py

```diff
@@ -181,6 +183,16 @@ def update_fits_wcs(self, model):
         """
         Update FITS WCS keywords of the resampled image.
         """
+        # Delete any SIP-related keywords first
+        pattern = r"(cd[123]_[123]|ap?_|bp?_)"
```

Comment:
this will delete anything with 'a_', 'b_' in it, so maybe make this a bit more strict to avoid deleting any future keywords that might contain these strings.

Reply:
Thanks.  Yeah, let me make this much more strict.